traffic_ops/traffic_ops_golang/login/login.go

```diff
@@ -483,17 +483,29 @@ func OauthLoginHandler(db *sqlx.DB, cfg config.Config) http.HandlerFunc {
 			return
 		}
 
-		decodedToken, err := jwt.Parse(
+		var decodedToken jwt.Token
+		if decodedToken, err = jwt.Parse(
 			[]byte(encodedToken),
 			jwt.WithVerifyAuto(true),
 			jwt.WithJWKSetFetcher(jwksFetcher),
-		)
-		if err != nil {
-			api.HandleErr(w, r, nil, http.StatusInternalServerError, nil, fmt.Errorf("Error decoding token with message: %w", err))
-			return
+		); err != nil {
+			if decodedToken, err = jwt.Parse(
+				[]byte(encodedToken),
+				jwt.WithVerifyAuto(false),
+				jwt.WithJWKSetFetcher(jwksFetcher),
+			); err != nil {
+				api.HandleErr(w, r, nil, http.StatusInternalServerError, nil, fmt.Errorf("Error decoding token with message: %w", err))
```

Comment:
I know this is not your code, but while you're in here, mind fixing the error statement to start with a lower case letter?